python/ray/serve/deployment_state.py

```diff
@@ -52,6 +53,13 @@ class ReplicaStartupStatus(Enum):
     FAILED = 4
 
 
+class HealthCheckResponse(Enum):
+    NONE = 1
+    SUCCEEDED = 2
+    FAILED = 3
```

Comment:
Can we change "Failed" to something like "Application unhealthy", and change "Actor crashed" to "Actor unhealthy"? It's not obvious what the difference between "Failed" and "Actor crashed" is based on just the enum names.